addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -1852,6 +1852,45 @@ QUnit.module('basic_fields', {
         form.destroy();
     });
 
+    QUnit.test('image fields in x2many list are loaded correctly', function (assert) {
+        assert.expect(3);
+
+        this.data.partner_type.fields.image = {name: 'image', type: 'binary'};
+        this.data.partner_type.records[0].image = 'product_image';
+        this.data.partner.records[0].timmy = [12];
+
+        var form = createView({
+            View: FormView,
+            model: 'partner',
+            data: this.data,
+            arch: '<form string="Partners">' +
+                    '<field name="timmy" widget="many2many">' +
+                        '<tree>' +
+                            '<field name="display_name"/>' +
+                            '<field name="image" widget="image"/>' +
+                        '</tree>' +
+                    '</field>' +
+                '</form>',
+            res_id: 1,
+            mockRPC: function (route, args) {
+                if (route === 'data:image/png;base64,product_image') {
+                    assert.step("The list's image should have been fetched");
```

Comment:
i think that a simple assert.ok(true, 'The list's image should have been fetched') would have been simpler here